hankel/hankel.py

```diff
@@ -517,19 +489,6 @@ def final_term_amplitude(cls, f, h, k=None, ndim=2):
             The amplitude of the final term in the sum.
         """
         if k is None:
-            return HankelTransform.G(f, h, k)
+            return HankelTransform.final_term_amplitude(f, h, k)
```

Comment:
Why is this part of this PR? But I don't have to strong opinions about it.

Reply:
Ah. It really shouldn't be here, I guess. I'll revert it.